### PR TITLE
Move Private Auction Actions to a better place

### PR DESCRIPTION
**Pull request by Talbatross**

### CODE REVIEW

File: assets/js/view/auction_round.rb [hunk 1]
```diff
@@ -20,7 +20,6 @@ def render
       h(:div, [
         h(UndoAndPass, undo: @game.actions.size.positive?),
         *render_companies,
-        render_input,
         h(View::Players, game: @game),
       ].compact)
     end
```
Comment (tobymao) [inline assets/js/view/auction_round.rb:25]: clean up action_round, remov selected_company, actionable and imports as they aren't used

File: assets/js/view/company.rb [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 module View
   class Company < Snabberb::Component
+    include Actionable
```
Comment (tobymao) [inline assets/js/view/company.rb:5]: require this to be safe

File: assets/js/view/company.rb [hunk 3]
```diff
@@ -86,14 +87,62 @@ def render
         ]),
       ]
 
-      if @bids
+      if @bids&.length&.positive?
```
Comment (tobymao) [inline assets/js/view/company.rb:90]: @bids&.any?

File: assets/js/view/company.rb [hunk 4]
```diff
@@ -86,14 +87,62 @@ def render
...
         children << h(:div, { style: bidders_style }, 'Bidders:')
         children << render_bidders
       end
 
       children << h(:div, { style: bidders_style }, "Owner: #{@company.owner.name}") if @company.owner
 
-      h(:div, props, children)
+      h(:div, [
```
Comment (tobymao) [inline assets/js/view/company.rb:97]: why do this, add it to the children instead

File: assets/js/view/company.rb [hunk 5]
```diff
@@ -86,14 +87,62 @@ def render
...
+        h(:div, props, children),
+        render_actions
+      ])
+    end
+
+    def render_actions
+      return h(:div) unless selected? && @game.round.auction?
```
Comment (tobymao) [inline assets/js/view/company.rb:104]: don't need this if you add it to the children

also don't use unless with multiple statements, change it to an if

File: assets/js/view/company.rb (hunk 4 above)
Comment (Talbatross) [reply assets/js/view/company.rb:97]: so that it's outside of the card.

File: assets/js/view/company.rb (hunk 2 above)
Comment (Talbatross) [reply assets/js/view/company.rb:5]: you mean add require 'view/actionable' on top?

File: assets/js/view/company.rb (hunk 4 above)
Comment (Talbatross) [reply assets/js/view/company.rb:97]: if I add to the children the action will appear within the card

File: assets/js/view/company.rb (hunk 2 above)
Comment (tobymao) [reply assets/js/view/company.rb:7]: yes

File: assets/js/view/company.rb (hunk 4 above)
Comment (tobymao) [reply assets/js/view/company.rb:99]: i see

File: assets/js/view/auction_round.rb [hunk 6]
```diff
@@ -1,17 +1,12 @@
...
 require 'view/undo_and_pass'
 
-require 'engine/action/bid'
-
 module View
   class AuctionRound < Snabberb::Component
-    include Actionable
-
-    needs :selected_company, default: nil, store: true
+    needs :game, store: true
```
Comment (tobymao) [inline assets/js/view/auction_round.rb:9]: does this need to be stored?

**Pull request merged**